deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -482,6 +481,8 @@ spec:
                   value: service-telemetry-operator
                 - name: ANSIBLE_GATHERING
                   value: explicit
+                - name: PROMETHEUS_WEBHOOK_SNMP_IMAGE
```

Comment:
This is not my change, but it came is as part of the bundle update. It must not have been updated previously after this was added to `build/operator.yaml`.

Reply:
Yea I noticed that I don't get the change unless I run it locally in the branch. I must have added it post-1.1 tag :(

It also didn't show up when I had `--output-dir` but worked when it was in the local repo. Thanks for adding that!